# HG changeset patch
# User Punarbasu Purkayastha <ppurka@gmail.com>
# Date 1327778894 -28800
# Node ID 819143fa0e1e8fd69180c47e98fe76fb2385ffe7
# Parent  78a0b42fab1f0ff75c350dac781905c3e3d61905
remove sage-hardcode_sage_root since it is no longer needed

diff --git a/sage-hardcode_sage_root b/sage-hardcode_sage_root
deleted file mode 100755
--- a/sage-hardcode_sage_root
+++ /dev/null
@@ -1,16 +0,0 @@
-#!/usr/bin/env python
-
-import sys
-
-file = sys.argv[1]
-
-destdir = sys.argv[2]
-
-F = open(file).read()
-i = F.find('.....')
-
-F = F[:i] + destdir + F[i+5:]
-
-open(file,'w').write(F)
-
-
